torch/distributed/fsdp/_common_utils.py

```diff
@@ -0,0 +1,23 @@
+from enum import auto, Enum
+
+
+class TrainingState(Enum):
```

Comment:
Since this is in a private file, we do not need to make this private. I should do this more consistently (e.g. some later PRs in this stack will violate this), but I will leave that as BE follow-ups.